packages/ckeditor5-editor-classic/src/classiceditor.ts

```diff
@@ -196,27 +206,28 @@ export default class ClassicEditor extends Editor {
 	 * If the initial data is passed, a detached editor will be created. In this case you need to insert it into the DOM manually.
 	 * It is available under the {@link module:editor-classic/classiceditorui~ClassicEditorUI#element `editor.ui.element`} property.
 	 *
-	 * @param {module:core/editor/editorconfig~EditorConfig} [config] The editor configuration.
-	 * @returns {Promise} A promise resolved once the editor is ready. The promise resolves with the created editor instance.
+	 * @param [config] The editor configuration.
+	 * @returns A promise resolved once the editor is ready. The promise resolves with the created editor instance.
 	 */
-	static create( sourceElementOrData, config = {} ) {
+	public static create( sourceElementOrData: HTMLElement | string, config: EditorConfig = {} ): Promise<Editor> {
```

Comment:
```suggestion
	public static create( sourceElementOrData: HTMLElement | string, config: EditorConfig = {} ): Promise<ClassicEditor> {
```

Reply:
That way, the caller would have access to ElementApi and DataApi of the promise result.